Classes/PackageFactory/Guevara/Domain/Model/Changes/Create.php

```diff
@@ -1,9 +1,19 @@
 <?php
 namespace PackageFactory\Guevara\Domain\Model\Changes;
 
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Service\NodeTypeManager;
+
 class Create extends AbstractCreate
 {
     /**
+     * TODO: find out why everything breaks if I remove this dependency
```

Comment:
Well because AbstractCreate does not have a nodeTypeManager property which is injected. You can/should move this property up to AbstractCreate...